trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name.strip.slice(0, 10) if @nameable.correct_name.to_s.length > 10
+  end
```

Comment:
- [ ] Please know that the function should only trip the name if it's longer than 10 characters, so you should have an else to your if, kindly add that.

Reply:
Hello 👋 . Thanks for your review. Please correct me if I'm wrong but as it trims the first 10 letters I suppose it would be always the same if letters are fewer. I mean there's no need for more code.

Reply:
Hello @AmaliaGM I think the logic you used is good and It is working as It should 😃 👍

Reply:
As a note, please remember to mention someone you are communicating with in the comment so that he/she can see the notification. Otherwise, they won't be aware of your comment 👍

Reply:
@BertrandConxy You're right .. Thanks for your advice will do!